addons/mail/static/src/discuss/discuss.scss

```diff
@@ -6,4 +6,33 @@
     .o-mail-discuss-actions button:not(.btn-secondary):hover {
         background-color: $gray-400;
     }
+
+    .o-mail-discuss-thread-name-container {
+        background-color: transparent;
+        height: 100%;
+        position: relative;
+
+        span {
+            height: 100%;
+            visibility: hidden;
+        }
+    
+        input {
+            background-color: transparent;
+            display: inline-block;
+            height: 100%;
+            left: 0;
+            position: absolute;
+            width: 100%;
+
+            &:focus {
+                background-color: $white;
+                border-color: $gray-400 !important;
+            }
+    
+            &:disabled {
+                color: $black;
+            }
+        }
```

Comment:
Could use style classname instead, e.g. `h-100`, `position-relative`, etc.

Reply:
What I thought was when I have to use a style sheet for a specific class, I would use the style sheet only so that when I want to change the style, I can adapt that easily. Otherwise, just use the bootstrap style classes only. Do you think we should use the style classes as many as possible even if it will be a mixing of style sheets and classes？

Reply:
Use classname whenever possible. SCSS as last resort